config/ftbquests/quests/chapters/natures_aura.snbt

```diff
@@ -932,9 +932,9 @@
 			y: -7.0d
 			subtitle: "Abaddon's Gate"
 			description: [
-				"Ender Chests with a twist. Ender Crates can be assigned to a channel in an Anvil by simultaneously renaming them and combining them with an Eye of Ender."
+				"Ender Chests with a twist. Ender Crates can be assigned to a channel in an Anvil by simultaneously renaming them and combining with an Eye of Ender."

```

Comment:
This isn't actually a typo though, I don't think, the original sounds correct.

Reply:
To be honest, for me it does sound a bit weird. But it is your choice which one is better :D